File: android/app/src/main/java/com/smf/mobile/etendo_app_loader/RNBootSplashActivityEtendo.java
package com.smf.mobile.etendo_app_loader;

import android.content.Intent;
import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class RNBootSplashActivityEtendo extends AppCompatActivity {

  @Override
  protected void onCreate(@Nullable Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    try {
      startActivity(new Intent(this, MainActivity.class));
      finish();
    } catch (Exception e) {
      e.printStackTrace();
    }
  }
}

File: android/app/src/main/java/com/smf/mobile/etendo_app_loader/RNBootSplashActivityEtendo.kt
package com.smf.mobile.etendo_app_loader

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class RNBootSplashActivityEtendo : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val intent = Intent(this, MainActivity::class.java)
        val extras = getIntent().extras
        if (extras != null) {
            intent.putExtras(extras)
        }
        startActivity(intent)
        finish()
    }
}
